Licensing/LICENSE.md

```diff
@@ -1,7 +1,8 @@
 The project without the build tools (Everything that is not a .patch file or in the buildSrc folder) are licensed under the MIT license found [here](MIT.md).<br>
 All files in the buildSrc folder are licensed under MIT found [here](../buildSrc/license.txt).<br>
-All patches (.patch files) marked with "lithium" are licensed under LGPL3 found [here](https://github.com/CaffeineMC/lithium-fabric/blob/1.16.x/dev/LICENSE.txt).<br>
-All patches (.patch files) marked with "hydrogen" are licensed under LGPL3 found [here](https://github.com/CaffeineMC/hydrogen-fabric/blob/1.16.x/LICENSE.txt).<br>
+All patches (.patch files) marked with "lithium" are licensed under LGPLv3 found [here](https://github.com/CaffeineMC/lithium-fabric/blob/1.16.x/dev/LICENSE.txt).<br>
+All patches (.patch files) marked with "hydrogen" are licensed under LGPLv3 found [here](https://github.com/CaffeineMC/hydrogen-fabric/blob/1.16.x/LICENSE.txt).<br>
 All patches (.patch files) marked with "krypton" are licensed under MIT found [here](https://github.com/astei/krypton/blob/master/LICENSE).<br>
+All Tuinity patches (.patch files) are licensed under LGPLv3 by default but some are under MIT if in the description of the patch , it says ```This patch is licensed under the MIT license. See /licenses/MIT.md``` then it's under the MIT license, otherwise it's under LGPLv3,  the licenses can be found there: [LGPLv3](https://github.com/Spottedleaf/Tuinity/blob/master/licenses/GPL.md) | [MIT](https://github.com/Spottedleaf/Tuinity/blob/master/licenses/MIT.md).<br>
```

Comment:
All tuinity patches are licensed under LGPLv3, not MIT. And this info is covered below with links to licenses.

Reply:
Some Tuinity patches from other authors are from What i think under MIT , except if the author asked that they want them to be under LGPLv3 and not MIT , [here is an example patch](https://github.com/Spottedleaf/Tuinity/blob/master/patches/server/0012-Update-version-fetcher-repo.patch) under MIT